src/client.ts

```diff
@@ -556,6 +563,33 @@ in an environmental variable named FAUNA_SECRET or pass it to the Client\
     return partialClientConfig?.endpoint ?? env_endpoint ?? endpoints.default;
   }
 
+  #getLogger(partialClientConfig?: ClientConfiguration): LogHandler {
+    if (
+      partialClientConfig &&
+      "logger" in partialClientConfig &&
+      partialClientConfig.logger === undefined
+    ) {
+      throw new TypeError(`ClientConfiguration option logger must be defined.`);
+    }
+
+    if (partialClientConfig?.logger) {
+      return partialClientConfig.logger;
+    }
+
+    if (
+      typeof process !== "undefined" &&
+      process &&
+      typeof process === "object" &&
+      process.env &&
+      typeof process.env === "object"
+    ) {
+      const env_debug = parseDebugLevel(process.env["FAUNA_DEBUG"]);
+      return new ConsoleLogHandler(env_debug);
+    }
+
+    return new ConsoleLogHandler(LOG_LEVELS.ERROR);
```

Comment:
Are we saying we'd always be publishing console.log by default? That doesn't seem desirable.

Reply:
Good point. How about I add levels 5-CRITICAL and 6-NO_LOGGING and default to 6 instead of 4-ERROR.

This would align better with .NET, too https://github.com/fauna/fauna-dotnet/blob/f6142aa3adc431138c22b13b26cc6aba417c016b/Fauna/Util/Logger.cs#L55

Reply:
Added levels 5 and 6 and a `.fatal()` method to the logger interface. Default is now level 6.